src/main/target/common_defaults_post.h

```diff
@@ -332,3 +332,12 @@
 #define USE_INVERTER
 #endif
 #endif
+
+#ifdef USE_MCO
```

Comment:
I think these defaults are not needed anymore; pin is hardcoded in `drivers/mco.c`.

Reply:
true, I removed them.